shared/constants/platform-specific/index.native.tsx

```diff
@@ -282,27 +285,36 @@ const onChatClearWatch = async () => {
 
 const locationTaskName = 'background-location-task'
 let locationRefs = 0
-ExpoTaskManager.defineTask(locationTaskName, ({data, error}) => {
```

Comment:
It seems like this'll re-pick-up on start with newer expo, so instead we define it only if we're about to use it